docs/docs/migration-guide.mdx

```diff
@@ -221,19 +198,20 @@ Note that the previous parameters `fallback_nlu_action_name` and
 `deny_suggestion_intent_name` are no longer configurable and have the fixed values
 `action_default_fallback` and `out_of_scope`.
 
-Please see the [Fallbacks](./fallback-handoff.mdx#fallbacks) documentation for more
+See the [fallback](./fallback-handoff.mdx#fallbacks) documentation for more
 information.
 
-#### Migrating from the Form Policy
+### Forms
+
+As of version 2.0 the logic for [forms](./forms.mdx) has been moved from the
+Rasa SDK to Rasa Open Source to simplify implementation and make it easier to write
+action servers in other languages.
 
-As of Rasa Open Source 2.0 the logic for [Forms](./forms.mdx) has been moved from the
-Rasa SDK to Rasa Open Source to simplify implementations of custom action libraries.
-This means that you no longer need to use the `FormAction` when implementing custom
-actions with the Python SDK. Instead you can `FormValidationAction` to validate and
-request slots.
+This means that forms are no longer implemented using a `FormAction`, but instead
+defined in the domain. Any customisations around requesting slots or slot validation
```

Comment:
```suggestion
defined in the domain. Any customisations around slot validation
```
Requesting slots is handled via special actions, but not formvalidationactions

Reply:
we can link to sections in the forms docs on this

Reply:
requesting slots is relevant here, if you have a custom slot mapping. so i'm leaving it in